packages/venia-ui/lib/RootComponents/Category/categoryContent.js

```diff
@@ -90,19 +97,13 @@ const CategoryContent = props => {
 export default CategoryContent;
 
 CategoryContent.propTypes = {
-    sortControl: shape({
-        currentSort: shape({
-            setSortDirection: string,
-            sortAttribute: string
-        }),
-        setSort: func.isRequired
-    }),
     classes: shape({
         filterContainer: string,
         gallery: string,
         headerButtons: string,
         pagination: string,
         root: string,
         title: string
-    })
+    }),
+    sortProps: array
```

Comment:
Since this component needs to know a couple levels deep of how this prop is structured, it would be helpful to expand this propType and define `sortProps` here using `arrayOf` and `shape`.

Reply:
I try both `arrayOf` leads to error
`Failed prop type: Invalid prop `sortProps[1]` of type `function` supplied to `ProductSort`, expected `object`.`

Shap will not work because we need to use array is much better for renaming variables (destruction).

Reply:
Did you want to add a comment like we talked about instead?